auth/src/main/java/org/aerogear/auth/RoleType.java

```diff
@@ -0,0 +1,8 @@
+package org.aerogear.auth;
+
+/**
+ * Keycloak role types
+ */
+public enum RoleType {
+    REALM, CLIENT
```

Comment:
perhaps do a ctor ? 

https://github.com/aerogear/aerogear-unifiedpush-server/blob/f6fe1622c595a40f54b6eb1c4f5a643e2e133f9a/service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/health/Status.java#L24-L27

Reply:
I'm not really seeing the benefit of this?